static/css/addons-form.css

```diff
@@ -1,171 +1,204 @@
-  }
-
-  .addons-form .title {
-    font-size: 1.3rem;
-    display: block;
-    width: 12.8rem;
-    height: 3.6rem;
-    overflow: hidden;
-    margin: 0.5rem 0;
-  }
-  
-  .addons-form .btn-form-tools:hover:enabled, button:active:enabled {
-    background-color: #658196;
-  }
-  
-  .addons-form .btn-form-tools::-moz-focus-inner {
-    border: 0;
-  }
-
-  .addons-form .btn-form-tools {
-    width: 3rem;
-    height: 3rem;
-    border-radius: 50%;
-    padding: 0;
-    background-color: #597285;
-    color: #fff;
-    border: none;
-    background-size: contain;
-    background-repeat: no-repeat;
-    -moz-user-select: none;
-    -webkit-user-select: none;
-    -ms-user-select: none;
-    user-select: none;
-    outline: 0;
-    cursor: pointer;
-  }
-
-  .addons-form .btn-add {
-    background-image: url('../images/add_arrayfield.svg');
-  }
-
-  .addons-form .btn-remove {
-    background-image: url('../images/clear_arrayfield.svg');
-    position: absolute;
-    bottom: 1.5rem;
-    right: 1rem;
-  }
-
-  .addons-form .array-item-row {
-    display: grid;
-    grid-template-columns: repeat(6, 1fr);
-    grid-gap: 0.5rem;
-    grid-auto-rows: minmax(3rem, auto);
-  }
-
-  .addons-form .array-item-col {
-    grid-column: span 6;
-    grid-row: 1;
-  }
-
-  .addons-form .array-item-col-field {
-    grid-column-start: span 5;
-    grid-row: 1;
-  }
-
-  .addons-form .array-item-col-tool {
-    position: relative;
-    grid-column: 6;
-    grid-row: 1;
-  }
-
-  .addons-form .errors-field.hidden {
-    display: none;
-  }
-  
-  #addon-config-settings .addons-form .errors-title {
-    margin: 0 auto;
-    padding-bottom: 0.5rem;
-    color: #FFCC66;
-  }
-
-  #addon-config-settings .addons-form .errors-list {
-    transform: translateY(0%);
-    top: auto;
-    margin: 0 auto;
-    padding: 0rem;
-    padding-left: 1em;
-    text-indent: -1em;
-    list-style-position: inside;
-    color: #FFCC66;
-  }
-  
-  .addons-form .addon-config-button-apply {
-    background-size: 2rem;
-    background-repeat: no-repeat;
-    background-position: 0.5em;
-    padding-left: 4rem;
-    margin-right: .8rem;
-    background-image: url('../images/check-16.svg');
-  }
-  
\ No newline at end of file
+#addon-config-settings {
+  background: #305067;
+}
+
+.addons-form {
+  top: 50%;
+  color: #fff;
+  font-size: 1.6rem;
+  min-width: 30rem;
+  max-width: 40rem;
+  margin: 0 auto;
+}
+
+.addons-form legend {
+  display: block;
+  width: 100%;
+  padding: 0;
+  margin-bottom: 0.5rem;
+  font-size: 2rem;
+  line-height: inherit;
+  border: 0;
+  border-bottom: 1px solid #fff;
+}
+
+.addons-form fieldset {
+  min-width: 0;
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+.addons-form > p > .form-group {
+  padding-left: 0;
+}
+
+.addons-form > p > .form-group > fieldset > legend {
+  font-size: 2.5rem;
+  border-bottom: none;
+  padding-bottom: 2rem;
+}
+
+.addons-form > p > .form-group > fieldset > legend:before {
+  content: 'Configuring: ';
```

Comment:
Is it possible to do this in HTML instead?

Reply:
Not easily. The form is completely auto-generated, and the top-level header is generated the exact same way as every nested header.

Reply:
Okay, SGTM then